src/components/nfts/NftGrid/index.tsx

```diff
@@ -90,7 +90,7 @@ const NftGrid = ({
   onPreview,
 }: NftsTableProps): ReactElement => {
   const chainId = useChainId()
-  const linkTemplates = nftPlatforms[chainId]
+  const linkTemplates = nftPlatforms[chainId] || []
```

Comment:
It would be better to add Sepolia to the config tho. It has a block explorer doesn't it?

Reply:
I can check but makes sense to add it. I would still keep the fallback in case of future networks that we are adding.